Modules/Core/TestKernel/include/itkTestDriverIncludeBuiltInIOFactories.h

```diff
@@ -1,83 +0,0 @@
-/*=========================================================================
- *
- *  Copyright NumFOCUS
- *
- *  Licensed under the Apache License, Version 2.0 (the "License");
- *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0.txt
- *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- *
- *=========================================================================*/
-#ifndef itkTestDriverIncludeBuiltInIOFactories_h
-#define itkTestDriverIncludeBuiltInIOFactories_h
-
-// ImageIO
-#include "itkBioRadImageIOFactory.h"
-#include "itkBMPImageIOFactory.h"
-#include "itkGDCMImageIOFactory.h"
-#include "itkNiftiImageIOFactory.h"
-#include "itkGiplImageIOFactory.h"
-#include "itkJPEGImageIOFactory.h"
-#include "itkLSMImageIOFactory.h"
-#include "itkMetaImageIOFactory.h"
-#include "itkPNGImageIOFactory.h"
-#include "itkNrrdImageIOFactory.h"
-#include "itkTIFFImageIOFactory.h"
-#include "itkVTKImageIOFactory.h"
-#include "itkStimulateImageIOFactory.h"
-#include "itkTestDriverInclude.h"
-#include "itkObjectFactoryBase.h"
-
-// MeshIO
-#include "itkBYUMeshIOFactory.h"
-#include "itkFreeSurferAsciiMeshIOFactory.h"
-#include "itkFreeSurferBinaryMeshIOFactory.h"
-#include "itkGiftiMeshIOFactory.h"
-#include "itkOBJMeshIOFactory.h"
-#include "itkOFFMeshIOFactory.h"
-#include "itkVTKPolyDataMeshIOFactory.h"
-
-#include "itkTestDriverInclude.h"
-#include "itkObjectFactoryBase.h"
-
-#ifdef __EMSCRIPTEN__
-#  include <emscripten.h>
-#endif
-
-void
-ProcessArgumentsAndRegisterBuiltInFactories(int * ac, ArgumentStringType * av)
-{
-  // ImageIO
-  itk::ObjectFactoryBase::RegisterFactory(itk::BioRadImageIOFactory::New());
```

Comment:
I think the point of this was to test all the built-in IOs. I guess that its existence is now moot.

Reply:
Sounds good. Given that tests pass after removing `itkTestDriverIncludeBuiltInIOFactories.h`, we can rely on `itkTestDriverIncludeRequiredFactories.cxx` for a factory testing.